lib/http-context.js

```diff
@@ -304,6 +306,31 @@ HttpContext.prototype.invoke = function(scope, method, fn, isCtor) {
   var pipe = http && http.pipe;
   var pipeDest = pipe && pipe.dest;
```

Comment:
I am proposing to deprecate `http.pipe.dest` in favour of this new "returns stream" API. Thoughts?